core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -503,12 +503,12 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
       logWarning(msg)
     }
 
-    val executorOptsKey = "spark.executor.extraJavaOptions"
-    val executorClasspathKey = "spark.executor.extraClassPath"
-    val driverOptsKey = "spark.driver.extraJavaOptions"
-    val driverClassPathKey = "spark.driver.extraClassPath"
-    val driverLibraryPathKey = "spark.driver.extraLibraryPath"
-    val sparkExecutorInstances = "spark.executor.instances"
+    val executorOptsKey = EXECUTOR_JAVA_OPTIONS.key
+    val executorClasspathKey = EXECUTOR_CLASS_PATH.key
+    val driverOptsKey = DRIVER_JAVA_OPTIONS.key
+    val driverClassPathKey = DRIVER_CLASS_PATH.key
+    val driverLibraryPathKey = DRIVER_LIBRARY_PATH.key
+    val sparkExecutorInstances = EXECUTOR_INSTANCES.key
```

Comment:
It seems that `sparkExecutorInstances` is not used. Can we clean up unused variable in this PR?